wordle-web/pages/leaderboard.vue

```diff
@@ -0,0 +1,44 @@
+<template>
```

Comment:
I maybe missing it, but I don't see anywhere to be able view the leaderboard.

Reply:
Thank you, we got it added, still need to connect it to the database